api/handler/acl.go

```diff
@@ -1358,13 +1424,15 @@ func permissionToOperations(permission amazonS3Permission) []eacl.Operation {
 }
 
 func encodeObjectACL(log *zap.Logger, bucketACL *layer.BucketACL, bucketName, objectVersion string) *AccessControlPolicy {
+	account := user.NewFromScriptHash(bucketACL.Info.OwnerPublicKey.GetScriptHash()).String()
```

Comment:
Why not using the owner account? You no longer need the key.

Reply:
I didn't want to make strong steps with key usage. Replaced with `Owner`